syntax = "proto2";

package hercules.proto;


enum ErrorCode {
    ERROR_SUCCESS = 0;
    ERROR_UNKNOWN = 1;
    ERROR_INTERNAL = 2;
    ERROR_NOT_FOUND = 3;
    ERROR_INVALID_ARG = 4;
    ERROR_UNAVAILABLE = 5;
    ERROR_UNSUPPORTED = 6;
    ERROR_ALREADY_EXISTS = 7;
}